stingray/multitaper.py

```diff
@@ -0,0 +1,441 @@
+import copy
+import warnings
+
+import numpy as np
+import scipy
+import scipy.optimize
+import scipy.stats
+
+import stingray.utils as utils
+from stingray.crossspectrum import AveragedCrossspectrum, Crossspectrum
+from stingray.gti import bin_intervals_from_gtis, check_gtis
+from stingray.largememory import createChunkedSpectra, saveData
+from stingray.stats import pds_probability
+from stingray.utils import genDataPath
+from stingray.utils import genDataPath, rebin_data, rebin_data_log, simon
+
+from .events import EventList
+from .gti import cross_two_gtis
+from .lightcurve import Lightcurve
+
+from nitime.lazy import matplotlib_mlab as mlab
+from nitime.lazy import scipy_linalg as linalg
+from nitime.lazy import scipy_signal as sig
+from nitime.lazy import scipy_interpolate as interpolate
+from nitime.lazy import scipy_fftpack as fftpack
+
+import nitime.utils as utils
+from nitime.utils import tapered_spectra, dpss_windows
+import nitime.algorithms as tsa
+import nitime.utils as utils
+from nitime.viz import winspect
+from nitime.viz import plot_spectral_estimate
+
+# To support older versions of numpy that don't have tril_indices:
+from nitime.index_utils import tril_indices, triu_indices
+
+try:
```

Comment:
Try to avoid duplicating lines by copy/paste. You can directly import show_progrss from crossspectrum.py